js/privatebin.js

```diff
@@ -3413,8 +3421,10 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
             TopNav.hideAllButtons();
             Alert.showLoading('Sending comment…', 0, 'cloud-upload');
 
-            // get data
-            var [plainText, nickname] = DiscussionViewer.getReplyData(),
+            // get data, note that "var [x, y] = " structures aren't supported in all JS environments
```

Comment:
They are supported [since](https://stackoverflow.com/questions/6983026/javascript-assign-array-values-to-multiple-variables?noredirect=1&lq=1)

    FireFox 2.0+
    IE 9
    Opera 11.50.

So if you really care to support these horribly ancient browsers, okay…
Otherwise the array assignment is likely more readable. Low-prio though…